docs/source/developers/guide/communication.rst

```diff
@@ -0,0 +1,77 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. _communication:
+
+*************
+Communication 
+*************
+
+**About the team**
+
+The team is full of experts, software engineers and core developers
+but also full of users, learners and enthusiasts that love doing what we
+do and we want to spread this enthusiasm to you also.
+
+We all have silly questions and we all need help lots of times.
+We encourage you to be open to communication and we will help as much as
+we can.
+
+Where to get help 👋
+====================
+
+Jira
+~~~~~~~~
+
+Thhe most used way is to communicate through Jira.
+When to use this option?
+
+- **Want to solve a bug** → post an issue on Jira and connect with the team
+  in the comments.
```

Comment:
If you want to _solve_ a bug, you might typically look for an existing JIRA, in which case "post an issue on Jira" -> "post a comment on Jira"? (edit: I see now this is the next bullet point ..)

Or could also rephrase as "Want to report a bug"

Reply:
Yeah, I see. Will rephrase. Thanks!